packages/taquito-http-utils/src/taquito-http-utils.ts

```diff
@@ -43,14 +43,14 @@ export class HttpRequestFailed implements Error {
 }
 
 export class HttpBackend {
-  private serialize(obj?: { [key: string]: any }) {
```

Comment:
Great work on this PR.

Just one question/comment: 
Would it have been possible to keep this method private?
I understand that you made it public to test it and that testing it via the `createRequest` method would not have been ideal. A hacky solution when I want to test a private method is to call it that way: `httpBackend['serialize'](httpRequestQuery)`
Let me know what you think.

Reply:
Thank you, I have made changes according to feedback.